src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -39,12 +39,12 @@ public ServiceManagerBuilder WithCallingAssembly()
         /// <returns>The instance of the <see cref="IServiceManager"/>.</returns>
         public IServiceManager Build()
         {
-            _options.ValidateOptions();
-
             var productInfo = ProductInfo.GetProductInfo(_assembly);
             var restClientBuilder = new RestClientFactory(productInfo);
 
-            return new ServiceManager(_options, productInfo, restClientBuilder);
+            return _options.InMultiEndpointState()
+                ? throw new NotImplementedException("multi-endpoint mode is not implemented yet")
```

Comment:
Make it a sentense `Multi-endpoint mode is not implemented yet.` and change `mode` to `service manager`, since we already has `mode` for `Persistent` and `Transient`